src/go/rdctl/pkg/snapshot/manager_windows_test.go

```diff
@@ -81,7 +84,7 @@ func TestManagerWindows(t *testing.T) {
 				t.Fatalf("failed to modify %s: %s", testFileName, err)
 			}
 		}
-		if err := manager.Restore(snapshot.ID); err != nil {
+		if err := manager.Restore(context.Background(), snapshot.Name); err != nil {
```

Comment:
I _think_ we may need to rebase this against `main` because the `snapshot.ID` change was already made?

Reply:
I think it was missed in a previous PR that should have changed it. I see that this line was fixed on November 22 by Eric. I'll rebase.